app/celery/scheduled_tasks.py

```diff
@@ -271,14 +271,14 @@ def check_for_services_with_high_failure_rates_or_sending_to_tv_numbers():
             len(services_with_failures)
         )
         for service in services_with_failures:
-            service_dashboard = current_app.config['ADMIN_BASE_URL'] + "/services/" + service.service_id
+            service_dashboard = current_app.config['ADMIN_BASE_URL'] + "/services/" + str(service.service_id)
```

Comment:
I think this is one reason why we prefer `str.format` over string concatenation, ie this works without doing any type casting:
```suggestion
            service_dashboard = "{}/services/{}".format(
                current_app.config['ADMIN_BASE_URL'],
                str(service.service_id),
            )
```